Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.8.1-onbuild
```

Comment:
*Shudder* a bit of overkill isn't it? have a look at this. https://github.com/CenturyLinkLabs/golang-builder